docs/installation.rst

```diff
@@ -82,9 +82,18 @@ Example app
 ===========
 
 There's an example application that showcases what django-import-export can do.
+It's assumed that you have set up a Python ``venv`` with all required dependencies
```

Comment:
I think it would be ideal to document the steps to setup the venv with the required dependencies.